### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by Snigdha17**

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.search-input {
+  width: 100%;
+  border: 1px solid var(--demo-color-1);
+  border-right: none;
+  padding: 0.5rem;
+  height: 4rem;
+  border-radius: 0.5rem 0 0 0.5rem;
+  outline: none;
```
Comment (MaruthiPothuganti) [inline css/navbar.css]: Outline is largely an accessibility feature and should be available on tabbable elements to indicate focus during keyboard navigation..It provides visual feedback for links that have "focus" when navigating a web document using the TAB key (or equivalent). This is especially useful for folks who can't use a mouse or have a visual impairment. If you remove the outline you are making your site inaccessible for these people.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <a href="../index.html" class="site-nav-logo">
+          <ion-icon name="cut"></ion-icon>
+          kustomfit
+        </a>
+        <div class="search-bar">
+          <input type="text" placeholder="Search" class="search-input" />
```
Comment (MaruthiPothuganti) [inline index.html]: input type of search can be used here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        </div>
+      </section>
+    </div>
+    <footer class="footer">
+      <p class="footer-text mg-bottom-xs">
+        Made with <ion-icon name="heart"></ion-icon> by Snigdha Kamal
+      </p>
+      <ul class="social-icons mg-bottom-xs">
+        <li>
+          <a class="social-link" href="https://github.com/Snigdha17"
```
Comment (MaruthiPothuganti) [inline index.html]: providing a target="_blank" is better for Social hyperlinks

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of changes and everything else looks great.

File: css/products.css [hunk 4]
```diff
@@ -0,0 +1,177 @@
+@import url("../style.css");
+
+.products-wrapper {
+  display: grid;
+  /* grid-template-columns: repeat(8, 1fr); */
```
Comment (dev-enforced) [inline css/products.css]: Can remove this commented line of code

File: css/navbar.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+.icon-btn .btn-icon {
+  height: 3.2rem;
+  width: 3.2rem;
+}
+
+.notif-badge {
+  width: 3.2rem;
+  height: 3.2rem;
+}
```
Comment (dev-enforced) [inline css/navbar.css]: These lines of code look repeated so you can make a common class and use it wherever needed

**Review by dev-enforced — APPROVED** [review]
The code looks good to me and the live preview also looks good!! Suggested some minor changes!! Do apply the changes given by @MaruthiPothuganti they are very insightful

File: css/navbar.css (hunk 1 above)
Comment (Snigdha17) [reply css/navbar.css]: Thank you Maruthi, I was not aware of this - will fix it right away

File: index.html (hunk 2 above)
Comment (Snigdha17) [reply index.html]: fixed it

File: index.html (hunk 3 above)
Comment (Snigdha17) [reply index.html]: done

File: css/products.css (hunk 4 above)
Comment (Snigdha17) [reply css/products.css]: done

File: css/navbar.css (hunk 5 above)
Comment (Snigdha17) [reply css/navbar.css]: good catch :) fixed it

**Pull request merged**